### ISSUE TITLE
neo4j 导入Nebula 

### ISSUE THREAD

**Issue by sataliulan** [vesoft-inc/nebula-graph#1161]

**Comment by wey-gu** [comment]
Dear @sataliulan,

Thank you for raising the issue here.
Have you accidentally removed the input of the issue, or?

**Comment by sataliulan** [comment]
Nope. Now , I am rebuilding the nebula-exchange following the WIKI(https://github.com/vesoft-inc/nebula-spark-utils/tree/master/nebula-exchange) , the mvn command is running at this moment.
I hope everything goes well.

**Comment by sataliulan** [comment]
By the way, I've read the nebular-spark-connector WIKI,  still, I don't know whether this connector has any relationship with nebular exchange ,or ?

**Comment by sataliulan** [comment]
> Dear @sataliulan,
> 
> Thank you for raising the issue here.
> Have you accidentally removed the input of the issue, or?

Hi，Wu， I've rebuilt the nebula-exchange, exec the cmd as : ./spark-submit --class com.vesoft.nebula.exchange.Exchange --master local ~/nebula-exchange-2.0-SNAPSHOT.jar -c ~/neo4j_to_nebula.conf

but I got the error still, as follows:
org.neo4j.driver.exceptions.ClientException: Database name parameter for selecting database is not supported in Bolt Protocol Version 3.0. Database name: 'graph.db'

So, any ideas?

**Comment by CPWstatic** [comment]
Would you please tell the details for this issue?

**Comment by wey-gu** [comment]
@sataliulan we will close this issue as I think it's continued via https://github.com/vesoft-inc/nebula-spark-utils/issues/99 and forum, right?
